ansible/cfgov_refresh_setup.yml

```diff
@@ -0,0 +1,17 @@
+---
+- hosts: db
+  sudo: yes
+  sudo_user: root
+  vars_files:
+    - vars/db.yml
+  roles:
+    - { role: geerlingguy.mysql }
+
+- hosts: app
+  sudo: yes
+  sudo_user: root
+  vars_files:
+    - vars/app.yml
+  roles:
+    - { role: geerlingguy.java }
+    - { role: vccabral.elasticsearch }
```

Comment:
points to my customized third party elasticsearch (https://github.com/vccabral/ansible-role-elasticsearch/tree/add_version)